From: Martin Schwidefsky <schwidefsky@de.ibm.com>
Subject: s390/hmcdrv: HMC drive CD/DVD access
Patch-mainline: no (kmsg not upstream)
References: bsc#934537,FATE#318066,LTC#KRN1015

Summary:     kernel: SE/HMC file access
Description: This feature provides a device driver that allows to access
             files residing on a HMC drive CD/DVD-ROM. It can be used in
             an LPAR or a z/VM environment.

Upstream-Description (kmsg not upstream):
             s390/hmcdrv: add kernel messages

Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 Documentation/kmsg/s390/hmcdrv |   21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

--- /dev/null
+++ b/Documentation/kmsg/s390/hmcdrv
@@ -0,0 +1,21 @@
+/*?
+ * Text: "Allocating the requested cache size of %zu bytes failed\n"
+ * Severity: Error
+ * Parameter:
+ *   @1: size
+ * Description:
+ * You cannot use the 'hmcdrv' module.
+ * Either the cache size that was specified for the 'hmcdrv' module exceeded
+ * the maximum of 1048576 (1 megabyte), or not enough free memory was
+ * available.
+ * If the 'hmcdrv' module was compiled into the kernel, the cache size was
+ * specified with the 'hmcdrv.cachesize' kernel parameter.
+ * For a separate 'hmcdrv' module, the cache size was specified with the
+ * 'cachesize=' module parameter.
+ * User action:
+ * Specify a smaller cache size and try again to load the module.
+ * Do not exceed the maximum specification of 1048576 (1 megabyte).
+ * If necessary, free some memory and try again.
+ * If the module is compiled into the kernel, you must reboot Linux to change
+ * the cache size specification.
+ */
